standard/stream_links.lua

```diff
@@ -87,13 +103,25 @@ function StreamLinks.processStreams(forwardedInputArgs)
 		)
 
 		if String.isNotEmpty(streamValue) then
+			local platform = platformName
+			local languageCode = 'en'
+			local count = 1
 			-- stream has no platform
-			if platformName ~= 'stream' then
-				local lookUpPlatform = StreamLinks.streamPlatformLookupNames[platformName] or platformName
+			if platform ~= 'stream' then
+				if StreamLinks.streamPlatformLookupNames[platform] then
+					count = tonumber(platform:match('(%d+)$'))
+					platform = StreamLinks.streamPlatformLookupNames[platform]
+				end
+
+				streamValue = StreamLinks.resolve(platform, streamValue)
+			end
 
-				streamValue = StreamLinks.resolve(lookUpPlatform, streamValue)
+			if FeatureFlag.get('new_stream_format') then
+				local key = StreamLinks._buildKey(platform, languageCode, count)
+				streams[key] = streamValue
+			else
+				streams[platformName] = streamValue
 			end
```

Comment:
```suggestion
			if FeatureFlag.get('new_stream_format') then
				local key = StreamLinks._buildKey(platform, languageCode, count)
				streams[key] = streamValue
			end
			streams[platformName] = streamValue
```
It may end up looking like this. Need to sync with API & L+
Liquipedia modules have been updated to handle both options.

Reply:
L+ needs it with backwards compatibility for the time being.